### PR TITLE
removed explanations of RBR vs SBR, we now support RBR

### PR DESCRIPTION
**Pull request by jvaidya**
Signed-off-by: jvaidya <jitendra.vaidya@gmail.com>

### CODE REVIEW

File: doc/VitessReplication.md [hunk 1]
```diff
@@ -3,70 +3,18 @@
...
-schema changes), while at the same time simplifying the replication stream (so
-slaves can be fast), by rewriting Update statements.
-
-Then, with statement based replication, it becomes easier to perform offline
-advanced schema changes, or large data updates. Vitess’s solution is called
-schema swap.
+row based. Vitess supports both these modes.
+
+For schema changes, if the number of affected rows is greater > 100k (configurable), we don't allow direct application
+of DDLs the recommended tool in such cases is gh-ost
```
Comment (sougou) [inline doc/VitessReplication.md]: DDLs. The recommend...

File: doc/VitessReplication.md [hunk 2]
```diff
@@ -3,70 +3,18 @@
...
 
-We plan to also support row based replication in the future, and adapt our tools
-to provide the same features when possible. See Appendix for our plan.
+When using statement based replication, Vitess helps by rewriting Update statements,
+therefore allowing complex schema changes, while at the same time simplifying the replication stream (so
+slaves can be fast). This is described in detail below.
```
Comment (sougou) [inline doc/VitessReplication.md]: Replace with:
Not all statements are safe for SBR: https://dev.mysql.com/doc/refman/8.0/en/replication-rbr-safe-unsafe.html. Vitess rewrites some of these statements to be safe for SBR, and others are explicitly failed.

File: doc/VitessReplication.md [hunk 3]
```diff
@@ -3,70 +3,18 @@
...
+
+Thus, with statement based replication, it becomes easier to perform offline
+advanced schema changes, or large data updates. Vitess’s solution is called
+schema swap (described below).
```
Comment (sougou) [inline doc/VitessReplication.md]: The schema  swap is independent of vitess's statement rewrites.

**Comment by derekperkins** [comment]
Are we recommending `gh-ost` over `pt-osc`?

**Comment by sougou** [comment]
Maybe recommend both?

**Pull request merged**